tests/samples/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10.11-bullseye
+
+# gcc and musl-dev needed for the pip install
+RUN apt update && apt install -y gcc musl-dev g++ libffi-dev libgit2-dev
+RUN pip install guarddog
```

Comment:
We probably need to install the local version (e.g. pip install .. or similar)

Reply:
Yep, I didn't add anything since it's mentioned in the README. Would you like something in particular?

Reply:
I think this should do a pip install of the current repo (similar to https://github.com/DataDog/guarddog/blob/main/Dockerfile#L6-L8)

Otherwise you're testing the already published GuardDog version, not the local one

Reply:
Yep, done in the `local_install_guarddog.sh`, so we'll be able to install guarddog from a specific Github branch.